acceptance/scripts/generic/testrun.sh

```diff
@@ -2,7 +2,7 @@
 set -x
 
 export GEM_SOURCE="https://artifactory.delivery.puppetlabs.net/artifactory/api/gems/rubygems/"
-export GENCONFIG_LAYOUT="${GENCONFIG_LAYOUT:-redhat6-64ma-ubuntu1404-64a-windows2008r2-64a}"
+export GENCONFIG_LAYOUT="${GENCONFIG_LAYOUT:-redhat6-64ma-ubuntu1604-64a-windows2012r2-64a}"
```

Comment:
This isn't necessary, I just question whether or not we still support those old OSes every time I look at this file, so I updated them. Happy to put it back if we think the previous versions were better for some reason.

Reply:
Ha, this is great. When I use this script I always update it to just be a single redhat7 vm (unless the tests I'm working on are platform specific or need a separate master and agent).

Reply:
Do we want to make it a smaller matrix? Though to be fair running Windows caught something for me here.

Reply:
I'm fine leaving it larger and having folks have to pare it down if they know they don't need those vms.